README.md

```diff
@@ -1,13 +1,29 @@
-#[Title here]
-#[Credits here]
+#Habbit Tracker
+
+#Khoa Cao
+#Susanna Stott
+#Briana Mason
+#Dhruv Parikh
```

Comment:
I think you need spaces between the # and the word here so that it will actually make the text bold ie `# Habit Tracker`.